torch/onnx/_internal/fx/_pass.py

```diff
@@ -163,6 +164,31 @@ def __init__(
         """
         self.module = module
         self.diagnostic_context = diagnostic_context
+        self.fake_mode = self._detect_fake_mode()
```

Comment:
missing docstring from `Args:` above.

Reply:
it's not an arg but I'll add it to instance attribute